java/ql/src/Security/CWE/CWE-022/TaintedPath.qhelp

```diff
@@ -7,26 +7,28 @@
 can result in sensitive information being revealed or deleted, or an attacker being able to influence
 behavior by modifying unexpected files.</p>
 
-<p>Paths that are naively constructed from data controlled by a user may contain unexpected special characters,
-such as "..". Such a path may potentially point anywhere on the file system.</p>
+<p>Paths that are naively constructed from data controlled by a user may be absolute paths or contain
+unexpected special characters, such as "..". Such a path may potentially point anywhere on the file system.</p>
```

Comment:
```suggestion
<p>Paths that are naively constructed from data controlled by a user may be absolute paths, or may contain unexpected special characters such as "..". Such a path could point anywhere on the file system.</p>
```

Minor: main suggestion here is just to use "could" instead of "may potentially" for emphasis and less redundancy (I realise "may potentially" was in the original, already-approved, text). Other than that I've just added some commas to break the sentence up a bit.

In any case, like all my suggestions, it's not urgent.